sql/files/defaultdata/r/run

```diff
@@ -43,10 +47,11 @@ fi
 # debugging.
 # export ONLINE_JUDGE=1 DOMJUDGE=1
 
+export TMPDIR=./
```

Comment:
If you know why this is necessary, please add a comment explaining the reason

Reply:
It was needed in the past but seems to now work without.